### PR TITLE
Update the hierarchy

### PR DESCRIPTION
**Pull request by Sameerthe-Big-O**

### CODE REVIEW

**Comment by Sameerthe-Big-O** [comment]
Aoa sir just i'll be adding the if else conditions for status code

File: config/env.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const environment = {
   port: process.env.PORT || 2022,
   nodeEnv: process.env.NODE_ENV || "development",
-  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/boilerplate",
+  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/HRMS",
```
Comment (AbubakarZorrain) [inline config/env.js:4]: Use Live db.

File: dsd [hunk 2]
```diff
@@ -0,0 +1,10 @@
+  create-hierarchy[m
```
Comment (AbubakarZorrain) [inline dsd:1]: What is this file?

File: routes/Hierarchy/controllers.js [hunk 3]
```diff
@@ -14,19 +14,15 @@ const controller = {
...
-      const addResponse = await hierarchy.get(req.params.id);
-      if (addResponse.message === "success") {
-        return httpResponse.SUCCESS(res, addResponse);
-      } else {
-        return httpResponse.UNAUTHORIZED(res, addResponse.data);
-      }
+  get: async (req, res) => {
+    const addResponse = await hierarchy.get(req.params.id);
+    if (addResponse.message === "success") {
+      return httpResponse.SUCCESS(res, addResponse);
```
Comment (AbubakarZorrain) [inline routes/Hierarchy/controllers.js:20]: Status Code?

File: services/hierarchy.js [hunk 4]
```diff
@@ -9,15 +9,18 @@ const hierarchySerive = {
...
-      if (savedData) {
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
+  get: async (id)=>{
+    try{
+        const data = await hierarchyModel.findById(id);
+        if(data){
+        return { message: "success", data };
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:16]: Do not send message from service layer. just send data. handle message in controller.

File: services/hierarchy.js [hunk 5]
```diff
@@ -31,26 +34,18 @@ const hierarchySerive = {
...
-    } catch (error) {
-      return { message: "error", data: error.message };
+    catch(error){
+       return { message: "error", data: error.message };
     }
-  },
+  }
 };
 
 export default hierarchySerive;
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:51]: Rename it correctly...

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

**Review by Sameerthe-Big-O — COMMENTED** [review]
please review

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]
Resolve Conflicts

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]
Resolve Conflicts...

**Pull request closed without merging**